src/utlist.h

```diff
@@ -600,6 +632,41 @@ do {
   }                                                                                            \
 } while (0)
 
+#define DL_INSERT_INORDER(head,add,cmp)                                                        \
+  DL_INSERT_INORDER2(head,add,cmp,prev,next)
+
+#define DL_INSERT_INORDER2(head,add,cmp,prev,next)                                                        \
+  do {                                                                                         \
+    LDECLTYPE(head) _pos;                                                                      \
+    _CASTASGN(_pos, head);                                                                     \
+    if (head) {                                                                                \
+      DL_LOWER_BOUND(_pos,add,cmp);                                                            \
+      DL_APPEND_ELEM(head,_pos,add);                                                           \
+    }                                                                                          \
+    else {                                                                                     \
```

Comment:
Whitespace nit: please cuddle this else as `} else {`. (Also please fix the misaligned `\` on line 638.)